test/Microsoft.AspNet.Mvc.TagHelpers.Test/FormTagHelperTest.cs

```diff
@@ -284,9 +302,15 @@ public class FormTagHelperTest
                                              attributes: new Dictionary<string, string>
                                              {
                                                  { "aCTiON", "my-action" },
-                                             },
-                                             content: string.Empty);
-            var context = new TagHelperContext(allAttributes: new Dictionary<string, object>(), uniqueId: "test");
+                                             });
+            var context = new TagHelperContext(
+                allAttributes: new Dictionary<string, object>()
+                {
+                    { "aCTiON", "http://www.contoso.com" }
+                },
```

Comment:
why doesn't an empty `allAttributes` dictionary work anymore?  nothing should be reading from the property in this scenario